microsoft-edge/visual-studio-code/elements-for-edge.md

```diff
@@ -0,0 +1,70 @@
+---
+description: How to use Elements for Microsoft Edge (Chromium) from VS Code
+title: Debug Microsoft Edge (Chromium) from VS Code
+author: erdraud
+ms.author: erdraud
+ms.date: 08/08/2019
+ms.topic: article
+ms.prod: microsoft-edge
+keywords: microsoft edge, web development, f12 tools, devtools, vs code, visual studio code, elements
+---
+
+# Elements for Edge VS Code extension
+
+By adding the [Elements for Edge](https://marketplace.visualstudio.com/items?itemName=ms-edgedevtools.vscode-edge-devtools) VS Code extension, you can use the browser's Elements tool from within Visual Studio Code. By either launching or attaching, the Elements tool will connect to an instance of Microsoft Edge, display the runtime HTML structure, and allow you to alter the layout or fix styling issues.
+
+![GIF of the Elements for Edge VS Code extension at work](./media/elements-for-edge.gif)
+
+## Launching Edge From the Elements tab
+
+![Elements for Edge tab](./media/elements-for-edge-logo.jpg) 
+
+Navigate to Elements (the icon shown above). Next to where it says Elements for Microsoft Edge: Targets, there is a plus sign that will open the browser for your app. If you selected the about:blank option, you will have to navigate to your web app in the browser for it to appear in the Elements panel in VS Code.
+
+## Launching Edge from the Debugger tab
+
+If you are accustomed to the Debugger tool, you can access Elements from that tool. Navigate to Debug (`Ctrl` + `Shift` + `D` on Windows or `Command` + `Shift` + `D` on Mac). 
```

Comment:
This first sentence is unclear to me. Do you mean Debugger as in the Debugger for Edge VS Code extension? If so, it would be good to link to that extension in the VS Marketplace. Or do you mean Debug as in the pane in VS Code that users can navigate to?